styles.css

```diff
@@ -129,6 +129,120 @@
     padding: 114px 24px;
     background-color: #e5e5e5;
     background-image: none;
+    position: relative;
+  }
+
+  .popup-card {
+    display: none;
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    right: 0;
+    left: 0;
+    overflow-y: scroll;
+    overflow-x: hidden;
+    backdrop-filter: blur(6px);
+    padding: 40px;
+  }
```

Comment:
### Your Design
![image](https://user-images.githubusercontent.com/93039359/224435713-66fa73d3-8829-4568-903f-c91cde0b1e06.png)

### Figma Design
![image](https://user-images.githubusercontent.com/93039359/224435744-f042b92b-8535-491e-86c4-c0146fd2f8b0.png)

- [ ] As suggested by the previous reviewer that it is also required to stick to the Figma design as much as possible. I would recommend you make your popup card to be as medium as possible for the desktop so that there wouldn't be any need for scrolling to see the rest of the information on the card. Kindly look at the Figma design and work your design alongside that 😊